src/frontend/qt_sdl/CMakeLists.txt

```
@@ -70,21 +69,41 @@ find_package(Threads REQUIRED)
 find_package(PkgConfig REQUIRED)
 find_package(Iconv REQUIRED)
 pkg_check_modules(SDL2 REQUIRED sdl2)
-pkg_check_modules(SLIRP REQUIRED slirp)
+pkg_check_modules(SLIRP slirp)
 pkg_check_modules(LIBARCHIVE REQUIRED libarchive)
 add_compile_definitions(ARCHIVE_SUPPORT_ENABLED)
 
+if (SLIRP_FOUND)
+    option(ENABLE_LIBSLIRP "Enable use of libslirp for network sockets" ON)
+else()
+    option(ENABLE_LIBSLIRP "Enable use of libslirp for network sockets" OFF)
+endif()
```

```suggestion
if (SLIRP_FOUND)
    option(ENABLE_LIBSLIRP "Enable use of libslirp for network sockets" ON)
    message(STATUS "libslirp found, and libslirp support is enabled")
else()
    option(ENABLE_LIBSLIRP "Enable use of libslirp for network sockets" OFF)
    message(WARNING libslirp not found, building without libslirp support!")
endif()
```
Add a generate time message so it's clear if `libslirp` is found or not.